doc/cephadm/octopus_gsg.rst

```diff
@@ -0,0 +1,262 @@
+        INFO:cephadm:Waiting for mgr to start...
+        INFO:cephadm:Waiting for mgr...
+        INFO:cephadm:mgr not available, waiting (1/10)...
+        INFO:cephadm:mgr not available, waiting (2/10)...
+        INFO:cephadm:Enabling cephadm module...
+        INFO:cephadm:Waiting for the mgr to restart...
+        INFO:cephadm:Waiting for Mgr epoch 5...
+        INFO:cephadm:Setting orchestrator backend to cephadm...
+        INFO:cephadm:Generating ssh key...
+        INFO:cephadm:Wrote public SSH key to to /etc/ceph/ceph.pub
+        INFO:cephadm:Adding key to root@localhost's authorized_keys...
+        INFO:cephadm:Adding host 192-168-1-113...
+        INFO:cephadm:Deploying mon service with default placement...
+        INFO:cephadm:Deploying mgr service with default placement...
+        INFO:cephadm:Deploying crash service with default placement...
+        INFO:cephadm:Enabling mgr prometheus module...
+        INFO:cephadm:Deploying prometheus service with default placement...
+        INFO:cephadm:Deploying grafana service with default placement...
+        INFO:cephadm:Deploying node-exporter service with default placement...
+        INFO:cephadm:Deploying alertmanager service with default placement...
+        INFO:cephadm:Enabling the dashboard module...
+        INFO:cephadm:Waiting for the mgr to restart...
+        INFO:cephadm:Waiting for Mgr epoch 13...
+        INFO:cephadm:Generating a dashboard self-signed certificate...
+        INFO:cephadm:Creating initial admin user...
+        INFO:cephadm:Fetching dashboard port number...
+        INFO:cephadm:Ceph Dashboard is now available at:
+
+        URL: https://192-168-1-113:8443/
+        User: admin
+        Password: 1nuk3cgx5m
+
+        INFO:cephadm:You can access the Ceph CLI with:
+
+        sudo /usr/sbin/cephadm shell --fsid 501812ec-a426-11ea-b71d-48f17fce7093 -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring
+
+        INFO:cephadm:Please consider enabling telemetry to help improve Ceph:
+
+        ceph telemetry on
+
+        For more information see:
+
+        https://docs.ceph.com/docs/master/mgr/telemetry/
+
+        INFO:cephadm:Bootstrap complete.
+
+   For more information on bootstrapping a Ceph cluster, see :ref:`cluster_bootstrap`.
+
+2. Enter the cephadm shell::
+
+        [root@192-168-1-113 zdover]# cephadm shell
+        INFO:cephadm:Inferring fsid c56ac3b8-a4cc-11ea-82a0-48f17fce7093
+        INFO:cephadm:Using recent ceph image docker.io/ceph/ceph:v15
+
+   .. note::
+            As of Octopus it is not necessary to install `cephadm-shell`. `cephadm-shell` is installed during the bootstrap procedure. If this doesn't mean anything to you, know that it used to be necessary at this point in the installation procedure to install a program called `cephadm-shell` so that you could interact with Ceph, but the Ceph release called "Octopus" automatically installs that program, making its explicit installation unnecessary.
```

Comment:
This needs to be wrapped and formatted properly. I don't think the proceeding context of "If this doesn't mean anything to you" is necessary, remove it. The initial statement is clear.